bbb-install.sh

```diff
@@ -306,6 +306,14 @@ HERE
 
   apt-get update
   apt-get dist-upgrade -yq
+  
+  # improve performance with low latency kernel, higher limits and larger networking buffer | issue #260
+  # roland.alton@fairkom.eu 2020-08-20 BBB hackathon
+  apt-get install linux-lowlatency linux-headers-lowlatency
+  echo -e "DefaultLimitNOFILE=65000\r\nDefaultLimitNPROC=65000\r\nDefaultTasksMax=65000" | sudo tee -a /etc/systemd/system.conf
```

Comment:
this line should be removed. Proper limits are already set in the services configurations. The way how it is done here would not work for current linux systems.